client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -420,6 +425,84 @@ export class DomainWarnings extends React.PureComponent {
 		);
 	};
 
+	onNewTransfersWrongNSNoticeClick = () => {
+		this.trackClick( newTransfersWrongNSWarning );
+	};
+
+	newTransfersWrongNS = () => {
+		const newTransfers = this.getDomains().filter(
+			domain =>
+				domain.registrationMoment &&
+				moment( domain.registrationMoment )
+					.add( 3, 'days' )
+					.isAfter( moment() ) &&
+				domain.transferStatus === transferStatus.COMPLETED &&
+				! domain.hasWpcomNameservers
+		);
+
+		if ( newTransfers.length === 0 ) {
+			return null;
+		}
+
+		const { translate } = this.props;
+		let compactMessage;
+		let actionLink;
+		let actionText;
+		let message;
+
+		if ( newTransfers.length > 1 ) {
+			actionLink = CHANGE_NAME_SERVERS;
+			actionText = translate( 'Learn more', {
+				comment: 'Call to action link for updating the nameservers on a newly transferred domain',
+			} );
+			compactMessage = translate( 'Domains require updating.' );
+			message = translate(
+				'To make your newly transferred domains work with WordPress.com, you need to ' +
+					'update the nameservers.'
+			);
+		} else {
+			const domain = newTransfers[ 0 ].name;
+			actionLink = domainManagementNameServers( this.props.selectedSite.slug, domain );
+			actionText = translate( 'Update now', {
```

Comment:
Can you make this just `Update` if it's a compact notice? Otherwise it looks off:
<img width="254" alt="screen shot 2018-04-10 at 13 29 10" src="https://user-images.githubusercontent.com/3392497/38554515-2d349682-3cc3-11e8-9aa2-f77731714f72.png">

Or maybe just `Fix`, to keep it consistent and prevent the notice from spilling into a new line:
<img width="256" alt="screen shot 2018-04-10 at 13 30 45" src="https://user-images.githubusercontent.com/3392497/38554586-7259282c-3cc3-11e8-93cb-bf250d31738a.png">